memo/_runner.py

```diff
@@ -45,6 +60,13 @@ def _run(self, func: Callable, settings: Iterable[Dict]) -> None:
         """run the parallel backend
         Private. All arguments passed through run method
         """
+        if self.backend == "ray":
+            try:
+                from ray.util.joblib import register_ray
+                register_ray()
+            except ImportError:
+                NotInstalled("ray", "dev")
+
```

Comment:
Is there a reason why we're not able to use `ray.init()` here?

Reply:
Yah we could do that. It would just require passing kwargs along to `ray.init()` through `Runner.run` then to `Runner._run` I thinks it's cleaner to let the user do `ray.init()` from their script / code.  But, that brings ups a question though. Is the try except block even necessary if the user performs `ray.init()` in their code because they'll receive an import error well beforehand. The argument I then see for putting `ray.init()` here is to assist users with a customized error message that will give them a little bit more help.

Reply:
My initial guess would be that the average user will have access to a laptop and not a compute cluster. Given that the main use-case will be merely a laptop it occurs to me as more plausible that the user doesn't want to be bothered with setting up a ray-cluster locally. Especially when using this package. 

We could support both, something like `ray` and `ray-cluster` as backend strings but I'm wondering if that over-complicates thing. If we had to assume one I'd favor the `ray.init()` route. Mainly because I imagine that's how many folks will want to use it. 

@bradday4 what are your thoughts?